src/main/java/Duke.java

```diff
@@ -1,10 +1,178 @@
+import java.util.Arrays;
+import java.util.Scanner;
+import java.util.ArrayList;
+
 public class Duke {
+    public static final String LINE = "\t__________________________________________________________";
+    public static final String[] COMMAND= {"todo", "deadline", "event", "done", "bye", "help"};
+
+    public static String[] getCommand(String inCommand) {
+        inCommand = inCommand.trim();
+        String[] temp = inCommand.split(" ");
+        return temp;
+    }
+
+    public static void throwException() {
+        System.out.println(LINE);
+        System.out.println("\t Input command is wrong. Enter \"help\" for list of accepted");
+        System.out.println("\t commands");
+        System.out.println(LINE);
+    }
+
+    public static void listCommands() {
+        System.out.println(LINE);
+        System.out.println("\t "+ Arrays.toString(COMMAND));
+        System.out.println(LINE);
+    }
+
+    public static void executeBye() {
+        System.out.println(LINE);
+        System.out.println("\t Bye.Hope to see you again soon!");
+        System.out.println(LINE);
+    }
+
+    public static void executeList(ArrayList<Task>l1) {
+        if(l1.isEmpty()) {
+            System.out.println(LINE);
+            System.out.println("\t Oops! No task has been assigned yet! Please enter a task");
+            System.out.println("\t before listing");
+            System.out.println(LINE);
+            return;
+        }
+        int i = 0;
+        System.out.println(LINE);
+        System.out.print("\t Here are the tasks in your list:\n");
+        for (i = 0; i < l1.size(); i++) {
+            int count = i + 1;
+            Task temp = l1.get(i);
+            System.out.println("\t " + count +"."+ temp.getTaskDescription().trim());
```

Comment:
Shouldn't the spaces between the string and operator be more consistent ?  :thinking:
System.out.println("\t " + count + "." + temp.getTaskDescription().trim());